module/epochs/qc_client.go

```diff
@@ -22,28 +21,31 @@ import (
 // TransactionSubmissionTimeout is the time after which we return an error.
 const TransactionSubmissionTimeout = 5 * time.Minute
 
+// QCFlowClientInterface to allow testing ...
+type QCFlowClientInterface interface {
```

Comment:
Can we generalize this as we'll use it for the similar DKG client as well. For example, move to `module` and call it `SDKClient` or `SDKClientWrapper`